sw/otbn/code-snippets/rsa_verify_3072.s

```diff
@@ -706,12 +405,35 @@ modexp_var:
   li        x8, 4
   bne       x2, x0, no_sub
   li        x8, 16
+
   no_sub:
 
-   /* store result in dmem starting at dmem[dptr_out] */
-  lw        x21, 28(x0)
-  loop      x30, 2
+  /* store result in output buffer */
+  addi      x21, x24, 0
+  loopi     12, 2
     bn.sid    x8, 0(x21++)
     addi      x8, x8, 1
 
   ret
+
+.data
+
+.globl out_buf
```

Comment:
For all the global data systems, can you add a line of documentation what these parameters are? That'll help software developers using this code from Ibex.